spec/views/shared/activities/activity_spec.rb

```diff
@@ -112,46 +116,91 @@
+              expect(rendered).to have_content(t("activerecord.attributes.activity.fstc_applies"))
+              expect(rendered).to have_content(t("activerecord.attributes.activity.covid19_related"))
+            end
+          end
 
-        context "when the activity is ODA" do
-          it "shows fields specific to ISPF ODA #{level}s" do
-            expect(rendered).to have_content(t("activerecord.attributes.activity.collaboration_type"))
-            expect(rendered).to have_content(t("activerecord.attributes.activity.fstc_applies"))
-            expect(rendered).to have_content(t("activerecord.attributes.activity.covid19_related"))
+          context "when the activity is non-ODA" do
+            let(:activity) { build(factory_name, :ispf_funded, ispf_theme: 1, ispf_partner_countries: ["IN"], is_oda: false) }
+
+            it "doesn't show fields irrelevant to ISPF non-ODA #{level}s" do
+              expect(rendered).not_to have_content(t("activerecord.attributes.activity.objectives"))
+              expect(rendered).not_to have_content(t("activerecord.attributes.activity.benefitting_countries"))
+              expect(rendered).not_to have_content(t("activerecord.attributes.activity.gdi"))
+              expect(rendered).not_to have_content(t("activerecord.attributes.activity.collaboration_type"))
+              expect(rendered).not_to have_content(t("activerecord.attributes.activity.sustainable_development_goals"))
+              expect(rendered).not_to have_content(t("activerecord.attributes.activity.aid_type"))
+              expect(rendered).not_to have_content(t("activerecord.attributes.activity.fstc_applies"))
+              expect(rendered).not_to have_content(t("activerecord.attributes.activity.policy_marker_gender"))
+              expect(rendered).not_to have_content(t("activerecord.attributes.activity.policy_marker_climate_change_adaptation"))
+              expect(rendered).not_to have_content(t("activerecord.attributes.activity.policy_marker_climate_change_mitigation"))
+              expect(rendered).not_to have_content(t("activerecord.attributes.activity.policy_marker_biodiversity"))
+              expect(rendered).not_to have_content(t("activerecord.attributes.activity.policy_marker_desertification"))
+              expect(rendered).not_to have_content(t("activerecord.attributes.activity.policy_marker_disability"))
+              expect(rendered).not_to have_content(t("activerecord.attributes.activity.policy_marker_disaster_risk_reduction"))
+              expect(rendered).not_to have_content(t("activerecord.attributes.activity.covid19_related"))
+              expect(rendered).not_to have_content(t("activerecord.attributes.activity.channel_of_delivery_code"))
+              expect(rendered).not_to have_content(t("activerecord.attributes.activity.oda_eligibility"))
+              expect(rendered).not_to have_content(t("activerecord.attributes.activity.oda_eligibility_lead"))
+            end
```

Comment:
Shouldn't these lists be the same, with the only difference being `to` or `not_to`? I think this is outside the scope of this PR though - looks like this change is just moving things around